lm_eval/tasks/hendrycks_test.py

```diff
@@ -103,16 +102,16 @@ def __init__(self):
 
 
 class GeneralHendrycksTest(MultipleChoiceTask):
-    VERSION = 0
-    DATASET_PATH = "hendrycks_test"
+    VERSION = 1
+    DATASET_PATH = "cais/mmlu"
```

Comment:
Why are we making this change?

Reply:
The dataset's name was changed on HF (much like the Pile ownership being transferred to us.) I believe currently "hendrycks_test" redirects to "cais/mmlu" but that might change in future, so pointing to "cais/mmlu" directly seems preferable now!

Reply:
Exactly. Also the redirect worked for me on the web, but resulted in an exception when using with load_dataset().